check-swlib.sh

```diff
@@ -158,11 +154,31 @@ else
     echo "Found Ansible: $(type ansible-playbook)"
 fi
 
+declare -a CMD_ARRAY=()
+
+CMD_ARRAY+=($ANSIBLE_PLAYBOOK)
+CMD_ARRAY+=(-i "$INVENTORY_FILE")
+
+if [[ -n "$ANSIBLE_PARAMS" ]]; then
+  echo "Processing ANSIBLE_PARAMS string: [$ANSIBLE_PARAMS]"
+  CMD_ARRAY+=(-e "$ANSIBLE_PARAMS")
+fi
+
+# Add any passthrough arguments from the script command line
+if [[ "$#" -gt 0 ]]; then
```

Comment:
Is the conditional even necessary?  If $@ is empty, I don't think passing an empty element to the array will hurt anything.